draft-issuance.md

```diff
@@ -0,0 +1,169 @@
+```
+ZIP: 
+Title: Smooth Out The Block Subsidy Issuance
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+Status: Draft
+Category: Consensus
+Created: 2023-08-23
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”,
+and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+"Network upgrade" - to be interpreted as described in ZIP 200. [2]
+
+“Block Subsidy” - the algorithmic issuance of ZEC on block creation – part of
+the consensus rules. Split between the miner and the Dev Fund. Also known as Block Reward.
+
+“Issuance” - The method by which unmined or unissued ZEC is converted to ZEC available
+to users of the network
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZSF_BALANCE(h)`” is the total ZEC available in the Zcash Sustainability Fund (ZSF),
+described in ZIP #TODO#. This is used to pay out Block Subsidies from at block height
+`h`, ie. “not yet mined ZEC at h”.
+
+“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
+
+"`ZsfBalanceAfter(h)`" is the total ZEC available in the ZSF _after_ the calculation
+involving the `BLOCK_SUBSIDY_FRACTION` is applied.
```

Comment:
I think this ZIP is going to be less confusing if it consistently uses `ZsfBalanceAfter(h)` (the function).

It isn't necessary to reference the node state `ZSF_BALANCE` (note the difference in capitalisation) in this ZIP, because it is defined in terms of the function in the other ZIP. (And we might remove or rename it in that ZIP.)

Other fixes:
2. There's an off-by-one error here, it's actually the balance after `h-1` that's used to pay out the subsidy in the next block. But that is too specific for the definitions, so it doesn't need to be repeated here.
3. The ZSF includes deposits.
4. "_after_ the calculation involving the `BLOCK_SUBSIDY_FRACTION` is applied" is ambiguous, and repeats what is specified later in the ZIP. (So it isn't needed here.)

```suggestion
“`ZsfBalanceAfter(h)`” is the total ZEC available in the Zcash Sustainability Fund (ZSF) after the transactions
in block `h`, described in ZIP #TODO reference#. The Sustainability Fund is used to pay out Block Subsidies
from unmined ZEC, and other fund deposits.

“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
```